clarity_ext/domain/analyte.py

```diff
@@ -34,11 +37,7 @@ def target_volume(self):
     @property
     def well(self):
         pos = PlatePosition.create(self.resource.location[1])
-        return Well(pos, self.plate)
-
-    @property
-    def container(self):
-        return self.resource.location[0]
```

Comment:
We have to rewrite this function later when we include tubes in our scripts as well. But maybe the proper name would be 'tube' then